cdap-security/src/test/java/co/cask/cdap/security/authorization/DummyAuthEnforce.java

```diff
@@ -45,14 +45,6 @@ public void testSingleAction(@Name("namespaceId") NamespaceId namespaceId) throw
       throw new EnforceNotCalledException();
     }
 
-    @Deprecated // tests that the presence of other annotations does not affect class rewrite
```

Comment:
@rohitsinha54 Can you confirm if this is safe to be removed.

Reply:
checked with @rohitsinha54, This is just to test presence of additional annotations, so clarified the note with additional information.